exonum/src/blockchain/schema.rs

```diff
@@ -51,24 +51,27 @@ define_names!(
 encoding_struct! (
     /// Configuration index.
     struct ConfigReference {
-        /// The height, starting from which this configuration becomes actual.
+        /// Height since which this configuration becomes actual.
         actual_from: Height,
         /// Hash of the configuration contents that serialized as raw bytes vec.
         cfg_hash: &Hash,
     }
 );
 
 encoding_struct! (
-    /// Transaction location in block.
+    /// Transaction location in a block.
+    /// The given entity defines the block where the transaction was
+    /// included and the position of this transaction in that block.
     struct TxLocation {
-        /// Height of block in the blockchain.
+        /// Height of the block where the transaction was included.
         block_height: Height,
-        /// Index in block.
+        /// Number designated to this transaction in the block.
         position_in_block: u64,
     }
 );
 
-/// Information schema for `exonum-core`.
+/// Information schema for `exonum core`. This structure provides helper
+/// functions for retrieving service information from the blockchain.
```

Comment:
What's meant by "service information"? This phasing may be confusing given that the schema is not attached to an (Exonum) service.